### PR TITLE
Update for current EventStore

### PR DESCRIPTION
**Pull request by markstur**
* Requires a non-empty sharding columns list (use pkey)
* Default port is now 1100.
* Fix some typos in comments

### CODE REVIEW

**Comment by markstur** [comment]
signed the CLA

File: lagom-eventstore-example/README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ To build and run this example, you need:
 - [git](https://git-scm.com/)
 - [Java SE 8 JDK](http://www.oracle.com/technetwork/java/javase/overview/index.html)
 - [Maven 3.2.1+](https://maven.apache.org/) to build and run the Lagom project (3.5.0 recommended)
-- IBM Db2 Event Store [downloaded](https://github.com/IBMProjectEventStore/EventStore-DeveloperPreview/releases/tag/1.1.1), [installed](https://www.ibm.com/support/knowledgecenter/SSGNPV/eventstore/desktop/welcome.html) and running on your local machine. (This process requires 12+Gb of downloads)
+- IBM Project EventStore [downloaded](https://github.com/IBMProjectEventStore/EventStore-DeveloperEdition/releases), [installed](https://www.ibm.com/support/knowledgecenter/SSGNPV/eventstore/desktop/welcome.html) and running on your local machine. (This process requires 12+Gb of downloads)
```
Comment (TimMoore) [inline lagom-eventstore-example/README.md]: This link label was changed from "IBM Project EventStore" to "IBM Db2 Event Store" by @stevemart in #31. Did you intend to change it back or was that a merge conflict?

File: lagom-eventstore-example/lagom-eventstore-impl/src/main/resources/ibm-event-store.conf [hunk 2]
```diff
@@ -3,27 +3,28 @@ ibm.eventstore {
 
   //
   // 'endpoints' is a comma-separated list of 'hostname:port' pairs. For example:
-  //    endpoints =  "192.168.1.33:5555,192.168.1.35:5555"
+  //    endpoints =  "192.168.1.33:1100,192.168.1.35:1100"
   //
-  // If you are running IBM Project eventStore locally this will probably be a single IP and
-  // the port 5555. Use `ifconfig` or `ipconfig`(depending on your platform) to find your IP
+  // If you are running IBM Project eventStore locally this will probably be
```
Comment (stevemart) [inline lagom-eventstore-example/lagom-eventstore-impl/src/main/resources/ibm-event-store.conf]: you can update the wording here

File: lagom-eventstore-example/README.md (hunk 1 above)
Comment (stevemart) [reply lagom-eventstore-example/README.md]: @markstur maaaaark hai.

@TimMoore probably a merge conflict, i think.

Comment (markstur) [reply lagom-eventstore-example/README.md]: merge mistake

File: lagom-eventstore-example/lagom-eventstore-impl/src/main/resources/ibm-event-store.conf (hunk 2 above)
Comment (markstur) [reply lagom-eventstore-example/lagom-eventstore-impl/src/main/resources/ibm-event-store.conf]: good catch, done

**Comment by TimMoore** [comment]
Thanks again, @markstur and @stevemart!

**Pull request merged**